js/repl/types.js

```diff
@@ -80,6 +80,7 @@ export type PluginState = LazyLoadedState & {
 
 export type PluginStateMap = { [name: string]: PluginState };
 
+export type filename = "file.js";
```

Comment:
You don't need this type, I believe what this is saying is that a filename can only be equal to `file.js`, which isn't what we want.